test/prefer-async-await.js

```diff
@@ -0,0 +1,52 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-async-await';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const header = `const test = require('ava');\n`;
+const errors = [{
+	ruleId: 'prefer-async-await',
+	message: 'Prefer using async/await instead of returning a Promise.'
+}];
+
+ruleTester.run('prefer-async-await', rule, {
+	valid: [
+		header + 'test(t => { t.is(1, 1); });',
+		header + 'test(t => { foo(); });',
+		header + 'test(t => { return foo(); });',
+		header + 'test(t => { foo().then(fn); });',
+		header + 'test(t => { function foo() { return foo().then(fn); } });',
+		header + 'test(t => foo().then(fn));',
+		// This should be an error, needs improvement
```

Comment:
Can you make this a `TODO` comment or open an issue about it so we don't forget?